pay-api/tests/unit/services/test_statement.py

```diff
@@ -322,6 +321,114 @@ def test_get_monthly_interim_statement(session, admin_users_mock):
     assert monthly_invoices[0].invoice_id == monthly_invoice.id
 
 
+def test_interim_statement_settings_eft(db, session, admin_users_mock):
```

Comment:
Ken found some anomalies which was more prevalent when swapping payment methods within the same day.
Fix was for this issue and added a test.